nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name(name)
+    name.downcase.capitalize[0, 10].strip
+  end
+end
```

Comment:
- [ ] The method 'correct_name' in the Naneable class should be rasing an error.